src/merge_sort.py

```diff
@@ -0,0 +1,72 @@
+"""Merge sort a list."""
+import time
+from random import randint
+
+
+def merge_sort(list_in):
+    """Sort a list using the merge sort method."""
+    if len(list_in) > 1:
+        half = len(list_in) // 2
+        list_a = list_in[0:half]
```

Comment:
You don't need to start this with a 0. If you provide no integer for the first argument of the slice, it'll start at 0 by default